docs/reverse_proxy.md

```diff
@@ -119,8 +130,9 @@ frontend https
   # Matrix client traffic
   acl matrix-host hdr(host) -i matrix.example.com
   acl matrix-path path_beg /_matrix
+  acl synapse-client-path path_beg /_matrix
```

Comment:
the path is wrong here, but I'd suggest combining them into one ACL rather than adding a new one:

```suggestion
  acl matrix-path path_beg /_matrix
  acl matrix-path path_beg /_synapse/client
```

(this will  make either path satisfy the ACL)